src/AWS.Deploy.CLI/Commands/CommandFactory.cs

```diff
@@ -258,6 +258,10 @@ private Command BuildDeployCommand()
                         _toolInteractiveService.WriteErrorLine(e.Message);
                     }
 
+                    _toolInteractiveService.WriteErrorLine(string.Empty);
+                    _toolInteractiveService.WriteErrorLine("For more information, please visit our troubleshooting guide https://aws.github.io/aws-dotnet-deploy/troubleshooting-guide/missing-dependencies/.");
```

Comment:
Seems weird to send all error messages to `missing-dependencies`. I recognize that is the nature of our doc site given you can't just click on the `Trouble Shooting` node. I wonder if we should rename the `other-issues` page to `general-issues`, make that page the first in the `Trouble Shooting` pages and then link to that page.

Reply:
Done